circle.yml

```diff
@@ -3,7 +3,7 @@ machine:
   java:
     version: oraclejdk8
   node:
-    version: 5.2.0
+    version: 6.9.0
```

Comment:
Was there a particular need to bump the version?

Reply:
Yes, the latest version of master from ~25 days ago  had stopped building on circe ci (and was never deployed). Changing the node version fixes this.